filters/builtin/loopbackifstatus.go

```diff
@@ -0,0 +1,63 @@
+package builtin
+
+import "github.com/zalando/skipper/filters"
+
+type loopbackIfStatusSpec struct {
+}
+type loopbackIfStatusFilter struct {
+	statusCode int
+	path       string
+}
+
+// NewLoopbackIfStatus Creates a filter spec for the loopbackIfStatus() filter.
+//
+//	r: * -> loopbackIfStatus(401, "/new-path") -> "https://www.example.org";
+//
+// It accepts two arguments: the statusCode code to and the path to change the request to.
+//
+// The filter replaces the response coming from the backend or the following filters.
+func NewLoopbackIfStatus() filters.Spec {
+	return &loopbackIfStatusSpec{}
+}
+
+func (c *loopbackIfStatusFilter) Request(ctx filters.FilterContext) {
+
+}
+
+func (c *loopbackIfStatusFilter) Response(context filters.FilterContext) {
+	if context.Response().StatusCode == c.statusCode {
+		context.Request().URL.Path = c.path
+		context.LoopbackWithResponse()
+	}
+}
+
+func (s *loopbackIfStatusSpec) CreateFilter(args []interface{}) (filters.Filter, error) {
```

Comment:
Please move this to the top before filter methods like we have it for all other filters.